File: control-assurance-sp/src/types/GIAAPeriod.ts
import { IEntity } from "./Entity";

export interface IGIAAPeriod extends IEntity {
    PeriodStatus?: string;
    PeriodStartDate?: Date;
    PeriodEndDate?: Date;
}
export class GIAAPeriod implements IGIAAPeriod {
    public ID: number = null;
    public Title: string = null;
    public PeriodStatus = null;
    public PeriodStartDate = null;
    public PeriodEndDate = null;
}

File: ControlAssuranceAPI/Models/GIAAPeriod.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class GIAAPeriod
{
    public int ID { get; set; }

    public string Title { get; set; } = null!;

    public string? PeriodStatus { get; set; }

    public DateTime? PeriodStartDate { get; set; }

    public DateTime? PeriodEndDate { get; set; }

    public int? LastPeriodId { get; set; }
}
